apps/model_eval_viz/app.py

```diff
@@ -207,10 +233,17 @@ def display_points(state_list:list,ids:str, data_set:str)->gv.Points:
     if ids:
         # filter stream_gage to only include the specified IDs
         id_list = [pid.strip() for pid in ids.split(",")]
+
+        site_no = id_list[0]
+        dates = (start_date.value,end_date.value)
+        qobs = nwis.get_streamflow(site_no, dates)
+        print(qobs)
+
```

Comment:
For calling the data from NWIS, we want this to be in a function outside of the function for plotting points.

Reply:
We will be using the data for a graph comparing streamflow to the modeled results and not graphing it on the points first then maybe adding flow graphs later.